apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerConstants.java

```diff
@@ -163,6 +163,26 @@
    */
   static final String CODING_NPI_US = "http://hl7.org/fhir/sid/us-npi";
 
+  /** The Unique Physician Identification Number (UPIN), as available at <a */
+  static final String CODING_UPIN = "https://bluebutton.cms.gov/resources/variables/upin";
+
+  /**
+   * The National Council for Prescription Drug Programs (NCPDP) provider identifier, as available
+   * at <a
+   */
+  static final String CODING_NCPDP = "https://bluebutton.cms.gov/resources/variables/ncpdp";
+
+  /** The State License Number as available at <a */
+  static final String CODING_STATE_LICENSE_NUMBER =
+      "https://bluebutton.cms.gov/resources/variables/state_license_number";
+
+  /** The Federal Tax Number as available at <a */
+  static final String CODING_FEDERAL_TAX_NUMBER =
+      "https://bluebutton.cms.gov/resources/variables/federal_tax_number";
```

Comment:
Have we asked the FHIR community if they want to define standard identifier `system`s for any of these? If we haven't, we should before we proceed, and we should note the FHIR tracker links to the requests/tickets.